<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtMCP Test Application</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="formTab">
       <attribute name="title">
        <string>Form</string>
       </attribute>
       <layout class="QVBoxLayout" name="formLayout">
        <item>
         <layout class="QFormLayout" name="inputFormLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="nameLabel">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="nameEdit">
            <property name="objectName">
             <string>nameEdit</string>
            </property>
            <property name="placeholderText">
             <string>Enter your name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="emailLabel">
            <property name="text">
             <string>Email:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="emailEdit">
            <property name="objectName">
             <string>emailEdit</string>
            </property>
            <property name="placeholderText">
             <string>Enter your email</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="messageLabel">
            <property name="text">
             <string>Message:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QTextEdit" name="messageEdit">
            <property name="objectName">
             <string>messageEdit</string>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>Enter your message</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="categoryLabel">
            <property name="text">
             <string>Category:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="comboBox">
            <property name="objectName">
             <string>comboBox</string>
            </property>
            <item>
             <property name="text">
              <string>General</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Support</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Feedback</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bug Report</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="checkBox">
            <property name="objectName">
             <string>checkBox</string>
            </property>
            <property name="text">
             <string>Subscribe to newsletter</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="priorityLabel">
            <property name="text">
             <string>Priority:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <layout class="QHBoxLayout" name="sliderLayout">
            <item>
             <widget class="QSlider" name="slider">
              <property name="objectName">
               <string>slider</string>
              </property>
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>50</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="sliderValueLabel">
              <property name="text">
               <string>50</string>
              </property>
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="buttonLayout">
          <item>
           <widget class="QPushButton" name="submitButton">
            <property name="objectName">
             <string>submitButton</string>
            </property>
            <property name="text">
             <string>Submit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearButton">
            <property name="objectName">
             <string>clearButton</string>
            </property>
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="resultGroup">
          <property name="title">
           <string>Result</string>
          </property>
          <layout class="QVBoxLayout" name="resultLayout">
           <item>
            <widget class="QTextEdit" name="resultText">
             <property name="objectName">
              <string>resultText</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="listTab">
       <attribute name="title">
        <string>List</string>
       </attribute>
       <layout class="QVBoxLayout" name="listLayout">
        <item>
         <widget class="QListWidget" name="listWidget">
          <property name="objectName">
           <string>listWidget</string>
          </property>
          <item>
           <property name="text">
            <string>Item 1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Item 2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Item 3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Item 4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Item 5</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tableTab">
       <attribute name="title">
        <string>Table</string>
       </attribute>
       <layout class="QVBoxLayout" name="tableLayout">
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="objectName">
           <string>tableWidget</string>
          </property>
          <property name="rowCount">
           <number>3</number>
          </property>
          <property name="columnCount">
           <number>3</number>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>150</number>
          </attribute>
          <row/>
          <row/>
          <row/>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Description</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>Alpha</string>
           </property>
          </item>
          <item row="0" column="1">
           <property name="text">
            <string>100</string>
           </property>
          </item>
          <item row="0" column="2">
           <property name="text">
            <string>First item</string>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>Beta</string>
           </property>
          </item>
          <item row="1" column="1">
           <property name="text">
            <string>200</string>
           </property>
          </item>
          <item row="1" column="2">
           <property name="text">
            <string>Second item</string>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string>Gamma</string>
           </property>
          </item>
          <item row="2" column="1">
           <property name="text">
            <string>300</string>
           </property>
          </item>
          <item row="2" column="2">
           <property name="text">
            <string>Third item</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
